src/test/java/org/broadinstitute/hellbender/utils/samples/SampleDBUnitTest.java

```diff
@@ -111,6 +111,15 @@ public void loadPEDString() {
         Assert.assertEquals(testPEDSamplesAsSet, db.getSamples());
     }
 
+    @Test()
+    public void loadPEDFromNames() {
+        List<String> names = new ArrayList<>();
+        testSAMSamples.forEach(s -> names.add(s.getID()));
+        builder.addSamplesFromSampleNames(names);
+        SampleDB db = builder.getFinalSampleDB();
+        Assert.assertEquals(testSAMSamples, db.getSamples());
```

Comment:
The actual value (`db.getSamples()`) should be the first argument to `assertEquals()`, and the expected value (`testSAMSamples`) should be the second argument.

Reply:
right

Reply:
also corrected same error in the pre-existing loadNonExistentFile()